modules/flowable-engine/src/main/java/org/flowable/engine/ProcessEngineConfiguration.java

```diff
@@ -157,6 +158,9 @@
     protected int cleanInstancesEndedAfterNumberOfDays = 365;
     protected HistoryCleaningManager historyCleaningManager;
 
+    // eventMessage
+    protected int eventMessageLength = 163;
+    protected int eventMessageSubLength = 163;
```

Comment:
Shouldn't this value be 160 and not 163 (to leave room for `...`)?

Reply:
ok i do it